src/app/views/directory.php

```diff
@@ -19,7 +19,12 @@
                     <?php if ($id == 0 && $user_can['see_global_trash']): ?>
                         <a href="<?= base_url() ?>trash/" class="btn btn-green"><i class="fa fa-trash"></i> Go to trash</a>
                     <?php endif; ?>
-                </div>
+		</div>
+		<?php 
+		    if (empty($data)) {
+			echo "<p>No files/directories are currently present, please check back later.</p>";
```

Comment:
This sounds like this is a blog and we're giving updates. :P There's no guarantee that someone will ever be putting up something, so the "check back later" part seems a little weird, don't you think?